webpack.config.js

```diff
@@ -2,6 +2,23 @@
 const path = require('path');
 // const CompressionPlugin = require("compression-webpack-plugin");
 
+class DtsBundlePlugin {
+  apply(compiler) {
+    compiler.hooks.done.tap(
+      'Dts bundle plugin',
+      stats => {
+        const dts = require('dts-bundle');
+
+        dts.bundle({
+          name: 'jsgeoda',
+          main: 'src/index.d.ts',
+          out: '../lib/index.d.ts'
```

Comment:
Use dts-bundle in webpack to merge multiple d.ts typings into `lib/index.d.ts` @dpeachpeach